proto/ibc/core/channel/v1/event.proto

```diff
@@ -0,0 +1,187 @@
+syntax = "proto3";
+package ibc.core.channel.v1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types";
+
+import "ibc/core/channel/v1/channel.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+
+// EventChannelOpenInit is a typed event emitted on channel open init
+message EventChannelOpenInit {
+    string port_id = 1;
```

Comment:
I think all field names with underscore are going to need yaml tags since yaml will default to camel case